src/Components/Councelors/index.js

```diff
@@ -72,7 +74,10 @@ const Councelor = () => {
         text="Are you sure you want to delete the counselor selected?"
       ></Modal>
       <ErrorMessage show={showErrorMessage} close={closeError} text={errorMessageText} />
-      <h2 className={styles.header}>Counselors</h2>
+      <div className={styles.titleAndButton}>
+        <h3>Councelors</h3>
```

Comment:
We should rename everything so we don't have this typo. But let's put it as tech debt
```suggestion
        <h3>Counselors</h3>
```